l10n_it_ddt/migrations/12.0.1.7.1/pre-migrate.py

```diff
@@ -1,14 +1,37 @@
 # Copyright 2020 Alex Comba - Agile Business Group
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
+import logging
 from openupgradelib import openupgrade
 
+_logger = logging.getLogger(__name__)
+
 _field_renames = [
     ('stock.picking.package.preparation', 'stock_picking_package_preparation',
      'show_price', 'ddt_show_price'),
 ]
 
 
+def delete_inherits_safely_by_id(env, xml):
+    for xml_child in xml.inherit_children_ids:
+        if xml_child.inherit_children_ids:
+            delete_inherits_safely_by_id(env, xml_child)
+        else:
+            delete_records_safely_by_id(env, xml_child)
+    delete_records_safely_by_id(env, xml)
+
+
+def delete_records_safely_by_id(env, xml_ids):
```

Comment:
Come mai non usare `openupgrade.delete_records_safely_by_xml_id`?

Reply:
Ho riscontrato dei problemi quando la vista è ereditata, però essendo che è normale che lo siano e non succede in giro chiudo.